qctrlopencontrols/driven_controls/driven_control.py

```diff
@@ -476,34 +513,70 @@ def export_to_file(
         file_type=FileType.CSV.value,
         coordinates=Coordinate.CYLINDRICAL.value,
     ):
-        """Prepares and saves the driven control in a file.
+        """
+        Prepares and saves the driven control in a file.
 
         Parameters
         ----------
-        filename : str, optional
+        filename : str
             Name and path of the file to save the control into.
-            Defaults to None
         file_format : str
```

Comment:
Sorry, didn't notice this in my first reading, but this might be optional

Reply:
Well spotted, done